src/cpp/session/modules/SessionEnvironment.R

```diff
@@ -735,3 +735,18 @@
    .Call("rs_hasAltrep", var, PACKAGE = "(embedding)")
 })
 
+.rs.addFunction("resolveContextSourceRefs", function(info)
+{
+   calls <- sys.calls()
+   
+   for (i in seq_along(calls))
+   {
+      fn <- sys.function(i)
+      if (identical(fn, info$callfun))
+         return(attr(calls[[i]], "srcref"))
```

Comment:
nit: `exact = TRUE`?

Reply:
Thanks for catching that -- fixed up now.